opal/mca/common/ucx/common_ucx_wpool.h

```diff
@@ -118,6 +120,7 @@ typedef struct {
  */
 typedef struct opal_common_ucx_winfo {
     opal_recursive_mutex_t mutex;
```

Comment:
```suggestion
    opal_list_item_t super; //?
    opal_recursive_mutex_t mutex;
```

Reply:
Why are we adding this? We're not storing the winfo as an opal list item. but instead we have a seperate winfo list item which includes the pointer to the winfo.